### ISSUE TITLE
promote the archives to central repo for fast downloading from nearest mirror

### ISSUE THREAD

**Issue by gzm55** [apache/maven-mvnd#707]
I know the mvnd official host location is https://downloads.apache.org/maven/mvnd/. If the binaries are also published to the central repo, we can download them from a faster local/internal mirror. The GAV could be:
```
group=org.apache.maven
artifact=maven-mvnd
classifier=linux-amd64/darwin-aarch64/generic/...
```

**Comment by gnodet** [comment]
You should be able to use `https://dlcdn.apache.org/maven/mvnd/0.8.2/` which uses the download CDN which should be fast enough.

**Comment by gzm55** [comment]
under https://dlcdn.apache.org/maven/mvnd/ there is only 0.8.2. Is previous versions, e.g. 0.8.1, deleted after a new version uploaded?

**Comment by gnodet** [comment]
> under https://dlcdn.apache.org/maven/mvnd/ there is only 0.8.2. Is previous versions, e.g. 0.8.1, deleted after a new version uploaded?

Yes, I think only the latest one is kept. Others can be downloaded from `https://archive.apache.org/dist/maven/mvnd/`.
